doc/source/examples/extended_examples/sfem/stochastic_fem.rst

```diff
@@ -0,0 +1,439 @@
+
+1. To generate the j-th realization, draw a random value for each :math:`\xi_{c,n}, n=1,\dots ,P, \quad \xi_{s,n}, n=1,\dots ,Q` from the standard
+   normal distribution :math:`\mathcal{N}(0,1)` and obtain :math:`\xi_{c,1}^j,\dots ,\xi_{c,P}^j, \quad \xi_{s,1}^j,\dots ,\xi_{s,P}^j`
+
+2. Insert these values into equation :math:numref:`approximation` in other to obtain the j-th realization:
+
+.. math:: \hat{X}^j(t) = \sum_{n=1}^P \sqrt{\lambda_{c,n}}\cdot\varphi_{c,n}(t)\cdot\xi_{c,n}^j + \sum_{n=1}^Q \sqrt{\lambda_{s,n}}\cdot\varphi_{s,n}(t)\cdot\xi_{s,n}^j
+
+3. To generate additional realizations, simply draw new random values for :math:`\xi_{c,n}, n=1,\dots ,P, \quad \xi_{s,n}, n=1,\dots ,Q` each from :math:`\mathcal{N}(0,1)`
+
+.. note::
+  In this case of a field, the preceding discussion applies as the only difference is a change in notation (for example :math:`t` to :math:`x`).
+
+The Monte Carlo simulation
+--------------------------
+For linear static problems in the context of FEM, the system equations which must be solved change from
+
+.. math:: \pmb{K}\pmb{U} = \pmb{F}
+
+to
+
+.. math:: \pmb{K}(\pmb{\xi})\pmb{U}(\pmb{\xi}) = \pmb{F}(\pmb{\xi})
+
+where :math:`\pmb{\xi}` collects a sources of system randomness. The Monte Carlo simulation for solving the preceding equation
+consists of generating a large number of :math:`N_{sim}` of samples :math:`\pmb{\xi}, i=1,\dots ,N_{sim}` from their probability
+distribution and for each of these samples, solving the deterministic problem
+
+.. math:: \pmb{K}(\pmb{\xi}_{(i)})\pmb{U}(\pmb{\xi}_{(i)}) = \pmb{F}(\pmb{\xi}_{(i)})
+
+The next step is to collect the :math:`N_{sim}` response vectors :math:`\pmb{U} := \pmb{U}(\pmb{\xi}_{(i)})` and perform a statistical
+post-processing to extract useful information such as mean value, variance, histogram,
+empirical pdf/cdf, etc.
+
+
+Problem description
+-------------------
+In the following plane stress problem
+
+.. figure:: problem.png
+
+   A two-dimensional cantilever structure under a point load
+
+:math:`P` is a random variable following the Gaussian distribution :math:`\mathcal{N}(0,1)` (kN) and the modulus of elasticity is a
+random field given by the expression:
+
+.. math:: E(x) = 10^5(1+0.10f(x)) (kN/m^2)
+
+with :math:`f(x)` being a zero mean stationary Gaussian field with unit variance. The covariance function for :math:`f` is
+
+.. math:: C_f(x_r,x_s)=e^{-\frac{\lvert x_r-x_s \rvert}{3}}
+
+The following are required:
+
+1. Using the K-L series expansion, generate 5000 realizations for :math:`E(x)` and perform Monte 
+   Carlo simulation to determine the probability density function of the response :math:`u`, at the bottom right corner 
+   of the cantilever. 
+
+2. If some design code stipulates that the displacement :math:`u` must not exceed :math:`0.2 \: m`, how confident can
+   one be that the structure meets this requirement?
```

Comment:
```suggestion
The following are required:

1. Using the K-L series expansion, generate 5000 realizations for :math:`E(x)` and perform Monte 
   Carlo simulation to determine the probability density function of the response :math:`u` at the bottom right corner 
   of the cantilever. 

2. If some design code stipulates that the displacement :math:`u` must not exceed :math:`0.2 \: m`, how confident can
   you be that the structure meets this requirement?
```
The following what are required? The second one is a question. I'm not sure the introduction is clear or appropriate. Also, are the points sequential? If not, a bulleted rather than a numbered list should be used.